package_check.sh

```diff
@@ -690,6 +700,49 @@ TEST_RESULTS () {
 		fi
 	fi
 
+	# Evaluate the eighth level
+	# -> High quality package.
+	# The level 8 can be validated only by the official list of app.
+	level[8]=0
+	# Define the level 8 only if we're working on a repository. Otherwise, we can't assert that this is the correct app.
+	if echo "$app_arg" | grep --extended-regexp --quiet "https?:\/\/"
+	then
+		# Get the last version of the app list
+		wget -nv https://raw.githubusercontent.com/YunoHost/apps/master/community.json  -O "$script_dir/list.json"
```

Comment:
This should be apps.json, if I well understood, community.json will no longer exists after that.

Reply:
Absolutely !
Probably a remaining of my tests, apps.json doesn't exist yet. So I was working on community.json.